CMakeLists.txt

```diff
@@ -500,9 +503,16 @@ if(BUILD_TESTS)
 
   if(ENABLE_JS_GOV)
     set(CONSTITUTION_ARGS
-        --constitution ${CCF_DIR}/src/runtime_config/default_actions.js
-        --constitution ${CCF_DIR}/src/runtime_config/test_actions.js
-        --constitution ${CCF_DIR}/src/runtime_config/constitution.js
+        --constitution
+        ${CCF_DIR}/src/runtime_config/default/actions.js
+        --constitution
+        ${CCF_DIR}/src/runtime_config/test/test_actions.js
```

Comment:
I had this as "actions.js" first, but for some bizarre reason, CLI11 seems to deduplicate it with `default/actions`. I will look at this later.

Reply:
It’s the copy in remote.py.